charts/akeyless-api-gateway/templates/deployment.yaml

```diff
@@ -266,6 +268,15 @@ spec:
                   name: {{ include "akeyless-api-gw.cacheSecretName" . }}
                   key: cache-pass
           {{- end }}
+          {{- if .Values.cachingConf.useClusterCache }}
+            - name: USE_CLUSTER_CACHE
+              value: "true"
+            - name: CACHE_ENCRYPTION_KEY
+              valueFrom:
+                secretKeyRef:
+                  name: {{ include "akeyless-api-gw.clusterCacheEncryptionKeyExist" . }}
```

Comment:
what happens if returns empty?

Reply:
the GW will print an error log, if the EncryptionKey is missing.